wpilibj/src/athena/cpp/lib/HALUtil.cpp

```diff
@@ -78,7 +80,11 @@ void ThrowHalHandleException(JNIEnv *env, int32_t status) {
   halHandleExCls.Throw(env, buf.c_str());
 }
 
-constexpr const char wpilibjPrefix[] = "edu.wpi.first.wpilibj";
+#ifndef _WIN32
+constexpr const char JNI_wpilibjPrefix[] = "edu.wpi.first.wpilibj";
+#else
+extern const char JNI_wpilibjPrefix[] = "edu.wpi.first.wpilibj";
```

Comment:
Oh... this is where we do the template thing that needs the length of the array.  Bah.